packages/@okta/vuepress-site/docs/reference/api/asa/attributes/index.md

```diff
@@ -0,0 +1,288 @@
+---
+title: Introduction to Attribute Endpoints
```

Comment:
The title of a page should just be the name of the API. For example "Attributes".